nautobot/core/settings.py

```diff
@@ -61,7 +61,7 @@
 
 # Disable linking of Config Context objects via Dynamic Groups by default. This could cause performance impacts
 # when a large number of dynamic groups are present
-CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED = os.getenv("CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED", "False")
+CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED = is_truthy(os.getenv("CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED", "False"))
```

Comment:
Please prefix the environment variable with `NAUTOBOT_`
```suggestion
CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED = is_truthy(os.getenv("NAUTOBOT_CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED", "False"))
```

Reply:
Updated